LEGO1/lego/legoomni/include/legohideanimpresenter.h

```diff
@@ -2,7 +2,8 @@
 #define LEGOHIDEANIMPRESENTER_H
 
 #include "decomp.h"
-#include "legoloopinganimpresenter.h"
+// TODO: Maybe this one is inlined as well?
```

Comment:
@foxtacles what do you think? I couldn't find any definitive info one way or the other regarding `LegoHideAnimPresenter`. Since all of the other presenters appear to be in `legoanimpresenter.cpp`, there's a good chance that this one is as well.

Reply:
yeah I'd probably move this one as well, seems consistent